packages/devtools_app/lib/src/flutter/table.dart

```diff
@@ -840,6 +840,7 @@ class _TableRowState<T> extends State<TableRow<T>>
         color: widget.backgroundColor ?? Theme.of(context).canvasColor,
         child: widget.onPressed != null
             ? InkWell(
+                canRequestFocus: false,
```

Comment:
does this prevent the row from being able to be selected?

Reply:
Oh, good question. You can still click on the headers to sort, and they have a slight highlight on mouseover. From a mouse-interaction perspective the behavior seems identical. But with my change you can't use tab or arrow keys to move the focus to the individual headers. It goes straight from the buttons like "Collapse All" to the table body.